.github/workflows/molecule.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           - rockylinux9
           - ubuntu2004
           - ubuntu2204
+          - ubuntu2404
         scenario:
           - ha
```

Comment:
And I would add `aio` to the scenario here so that it runs for all these operating systems.